### DISCUSSION TITLE
Bitcoin donations

### DISCUSSION THREAD

**Discussion by exquo** [AsamK/signal-cli#627] [Ideas]
Would you consider accepting donating in bitcoin? (In addition to the current card and bank transfers). I imagine this would be more convenient for a few of project's users.

**Comment by AsamK** [comment]
sure, I've added a donation address to the FUNDING.yml (though bitcoin is not supported by github there ...)
https://github.com/AsamK/signal-cli/blob/master/FUNDING.yml

**Reply by iBobik** [reply]
How about Lightning? I does not want to pay on-chain fees.

**Comment by exquo** [comment]
Great, thanks!

On Thu-10:44, AsamK wrote:
> sure, I've added a donation address to the FUNDING.yml (though bitcoin is not supported by github there ...)
> https://github.com/AsamK/signal-cli/blob/master/FUNDING.yml
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/AsamK/signal-cli/discussions/627#discussioncomment-764650

-------------------------------------------------
This free account was provided by VFEmail.net - report spam to ***@***.***
 
ONLY AT VFEmail! - Use our Metadata Mitigator to keep your email out of the NSA's hands!
$24.95 ONETIME Lifetime accounts with Privacy Features!  
15GB disk! No bandwidth quotas!
Commercial and Bulk Mail Options!
